Cargo.toml

```diff
@@ -14,6 +14,11 @@ authors = ["Niko Matsakis <niko@alum.mit.edu>"]
 license = "Apache-2.0 OR MIT"
 version = "0.19.9" # LALRPOP
 edition = "2021"
+# This is (very) soft limit of the minimum supported version.
+# Please update it when lalrpop requires a new feature.
+# We prefer to avoid the latest 2 stable versions. (e.g. at most 1.64 when stable is 1.66)
+# NOTE: Don't forget to update test.yaml as well.
+rust-version = "1.64.0"
```

Comment:
Sorry, I should have been more explicit last time.  I think this one could do with less specificity as well (just 1.64).

Reply:
Thank you! I fixed here too